transfers/owner_transfer.py

```diff
@@ -0,0 +1,95 @@
+# ===============================================================================
+# Copyright 2025 ross
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+import numpy as np
+import pandas as pd
+from transfers.util import read_csv, filter_to_valid_point_ids
+from db import Thing, Contact, ThingContactAssociation, Email, Phone, Address
+
+
+def transfer_owners(session):
+
+    odf = read_csv("ownersdata.csv")
+    odf = odf.replace(pd.NA, None)
+    odf = odf.replace({np.nan: None})
+
+    odf = filter_to_valid_point_ids(session, odf)
+    for i, row in odf.iterrows():
+        thing = session.query(Thing).where(Thing.name == row.PointID).first()
+        if thing is None:
+            print(f"Thing with PointID {row.PointID} not foaund. Skipping owner.")
+            continue
+
+        # TODO: put in guards for null values
+        contact1 = Contact(name=f"{row.FirstName} {row.LastName}", role="Primary")
```

Comment:
The roles are often defined in the `OwnerComment` field. It may need to be a manual process to correctly identify all of them. We could, however, look for key words